flink-end-to-end-tests/flink-end-to-end-tests-aws-kinesis-firehose/src/test/resources/send-orders.sql

```diff
@@ -29,7 +29,7 @@ CREATE TABLE orders (
   'aws.credentials.basic.secretkey' = 'secretAccessKey',
   'aws.trust.all.certificates' = 'true',
   'sink.http-client.protocol.version' = 'HTTP1_1',
-  'sink.batch.max-size' = '1',
+  'sink.batch.max-size' = '2',
```

Comment:
Why did you need to change this?

Reply:
Was testing the impact on the ITCase. Reverted